.babelrc

```diff
@@ -0,0 +1,24 @@
+{
+  "env": {
+    "development": {
```

Comment:
ideally these should be different, but for now we are using node 8 as a base, so I don't want to unnecessarily transpile to await and async. When/if we want to support early versions of node, or browser - we should update this.